### PR TITLE
feat: include oidc auth in tab

### PR DESCRIPTION
**Pull request by henryiii**
This became publicly available a couple of days ago.

### CODE REVIEW

File: pages/developers/gha_pure.md [hunk 1]
```diff
@@ -128,31 +128,68 @@ later in the upload action for the release job, as well).
...
+
 {% raw %}
 
 ```yaml
-publish:
+upload_all:
   needs: [dist]
+  permissions:
+    id-token: write
   runs-on: ubuntu-latest
```
Comment (webknjaz) [inline pages/developers/gha_pure.md:145]: It is also recommended to use GH environments as an extra security measure:
```suggestion
  runs-on: ubuntu-latest
  environment: pypi
```
(if an environment doesn't exist, GH will pre-create it)

File: pages/developers/gha_pure.md [hunk 2]
```diff
@@ -211,11 +307,12 @@ jobs:
...
-{%- endcapture -%}
 
-{{ mymarkdown | markdownify }}
+</div>
 
 </details>
 
 [pep 517]: https://www.python.org/dev/peps/pep-0517/
 [pep 518]: https://www.python.org/dev/peps/pep-0518/
+
```
Comment (matthewfeickert) [inline pages/developers/gha_pure.md:317]:
```suggestion
[PyPI trusted publisher docs]: https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/

```

The other half of my suggestion from above

File: pages/developers/gha_wheels.md [hunk 3]
```diff
@@ -175,23 +185,55 @@ upload_all:
...
 
-    - uses: pypa/gh-action-pypi-publish@v1.8.5
+    - uses: pypa/gh-action-pypi-publish@release/v1
+```
+
+{% endraw %}
+
+When you make a GitHub release in the web UI, we publish to PyPI. You'll just
+need to tell PyPI which org, repo, workflow, and set the `pypi` environment to
+allow pushes from from GitHub. If it's the first time you've published a
```
Comment (matthewfeickert) [inline pages/developers/gha_wheels.md:195]:
```suggestion
allow pushes from GitHub. If it's the first time you've published a
```

Fix of typo from earlier.

File: pages/developers/gha_pure.md [hunk 4]
```diff
@@ -128,31 +128,70 @@ later in the upload action for the release job, as well).
...
+    - uses: pypa/gh-action-pypi-publish@release/v1
+```
 
+{% endraw %}
+
+When you make a GitHub release in the web UI, we publish to PyPI. You'll just
+need to tell PyPI which org, repo, workflow, and set the `pypi` environment to
+allow pushes from from GitHub. If it's the first time you've published a
+package, go to
+https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/ for
```
Comment (matthewfeickert) [inline pages/developers/gha_pure.md:163]: Can actually link to it with some human readable text?

```suggestion
package, go to the [PyPI trusted publisher docs] for
```

and then at the bottom of the page with links

```markdown
[PyPI trusted publisher docs]: https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/
```

File: pages/developers/gha_wheels.md [hunk 5]
```diff
@@ -175,23 +185,55 @@ upload_all:
...
+package, go to
+https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/ for
```
Comment (matthewfeickert) [inline pages/developers/gha_wheels.md:197]:
```suggestion
package, go to the [PyPI trusted publisher docs] for
```

and then at the bottom of the page with links

```markdown
[PyPI trusted publisher docs]: https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/
```

File: pages/developers/gha_wheels.md [hunk 6]
```diff
@@ -203,3 +245,5 @@ Remember to set `pypi_password` to your token in secrets.
 [manylinuxgcc]: https://github.com/scikit-hep/manylinuxgcc
 [cibw custom]: https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
 [cibuildwheel]: https://cibuildwheel.readthedocs.io/en/stable/
+
```
Comment (matthewfeickert) [inline pages/developers/gha_wheels.md:248]:
```suggestion
[PyPI trusted publisher docs]: https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/

```

**Review by matthewfeickert — CHANGES_REQUESTED** [review]
@henryiii this LGTM beyond a typo fix and a suggestion on the PyPI docs URL.

Screenshot from 2023-04-22 02-10-34

Thanks for putting this together so fast!

**Pull request merged**